### PR TITLE
add  unit tests for mksTaks

### PR DESCRIPTION
**Pull request by ramessesii2**
## Main task
* unit test for create, delete, update, list, get methods

## Other fixes
* Add other actions in actions/
* add get, list, delete, update actions
* change kubeConfig default value in main.go
* read userHomeDir()/.kube/config as the default value

### How to test
1. Just fork and cd into dir
2. 
```bash
go test -v ./pkg/controllers/mkstask/ 
```

### CODE REVIEW

File: pkg/actions/get.go [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+	"context"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/discovery"
+	"k8s.io/client-go/dynamic"
+)
+
+func Get(gr schema.GroupVersionResource, dynamic dynamic.Interface, discovery discovery.DiscoveryInterface, objname, ns string, op metav1.GetOptions) (*unstructured.Unstructured, error) {
```
Comment (avinal) [inline pkg/actions/get.go]: Please change according to the rest of the actions

Comment (ramessesii2) [reply pkg/actions/get.go]: done

File: pkg/actions/get.go [hunk 2]
```diff
@@ -2,24 +2,17 @@ package actions
...
-	obj, err := dynamic.Resource(*gvr).Namespace(ns).Get(context.Background(), objname, op)
+func Get(gr schema.GroupVersionResource, clients *tconfig.Client, objname, ns string, opt metav1.GetOptions) (*unstructured.Unstructured, error) {
+	gvr, err := GetGroupVersionResource(gr, clients.Tekton.Discovery())
+	obj, err := clients.Dynamic.Resource(*gvr).Namespace(ns).Get(context.Background(), objname, opt)
 	if err != nil {
 		return nil, err
 	}
-
-	return obj, nil
+	return obj, err
```
Comment (avinal) [inline pkg/actions/get.go]:
```suggestion
	return obj, nil
```

**Review by UtkarshChaurasia — APPROVED** [review]
LGTM

File: pkg/actions/get.go (hunk 2 above)
Comment (ramessesii2) [reply pkg/actions/get.go]: okie

**Review by avinal — APPROVED** [review]

**Pull request merged**